scss/variables/typography.js

```diff
@@ -52,7 +52,7 @@ chassis.typography = {
 	},
 	"sans": {
 		name: "Font Family - Sans",
-		value: "Helvetica, Arial, sans-serif"
+		value: "Open Sans, Helvetica, Arial, sans-serif"
```

Comment:
Also a font that would have to be included.

Also in general this sort of typography change probably shouldn't be occurring within the scope of this PR - but something that should be brought up in a larger discussion.